index.bs

```diff
@@ -192,6 +192,7 @@ interface ImageCapture {
     readonly attribute MediaSettingsRange         imageHeight;
     readonly attribute MediaSettingsRange         imageWidth;
     readonly attribute FrozenArray&lt;FillLightMode> fillLightMode;
+    readonly attribute FrozenArray&lt;DOMString>     supportedImageFormats;
```

Comment:
@yellowdoge I noticed that impl status.md has 'flattened' features for PhotoSettings and PhotoCapabilities, because attributes have same names. Should I rename 'image format' related attributes as well?